runtime/doc/lua.txt

```diff
@@ -1645,4 +1645,25 @@ uri_to_fname({uri})                                       *vim.uri_to_fname()*
                 Return: ~
                     Filename
 
+
+==============================================================================
+Lua module: ui                                                        *lua-ui*
+
+select({items}, {opts}, {callback})                              *ui.select()*
```

Comment:
```suggestion
select({items}, {opts}, {callback})                              *vim.ui.select()*
```

Reply:
Should I also rename `callback` to `on_choice` here?

Reply:
i'd say yes. that is consistent with e.g. the callbacks in jobstart()